civihr_employee_portal/civihr_leave_absences/templates/my_leave.html

```diff
@@ -11,29 +11,574 @@
         <li><a href="#">Action 1</a></li>
       </ul>
     </div>
-  </div>
-  <div class="tabs-container">
-    <div class="container">
-      <ul class="nav nav-tabs nav-justified nav-tabs-header">
-        <li role="presentation" class="active">
-          <a href="#report">Report</a>
-        </li>
-        <li role="presentation">
-          <a href="#calendar">Calendar</a>
-        </li>
-      </ul>
-      <div class="tab-content">
-        <div id="report" class="tab-pane fade in active">
-          <h3>Report</h3>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-          </p>
-        </div>
-        <div id="calendar" class="tab-pane fade">
-          <h3>Calendar</h3>
-          <p>
-            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodoconsequat.
-          </p>
+    <div class="tabs-container">
+        <div class="container">
+            <ul class="nav nav-tabs nav-justified nav-tabs-header">
+                <li role="presentation" class="active">
+                    <a href="#report">Report</a>
+                </li>
+                <li role="presentation">
+                    <a href="#calendar">Calendar</a>
+                </li>
+            </ul>
+            <div class="tab-content">
+                <div id="report" class="tab-pane fade in active">
+
+                    <div class="container-fluid chr-container">
+                      <div class="row chr-container__row">
+                          <div class="col-md-9 ">
+                              <a href="#"><i class="fa fa-download" aria-hidden="true"></i><span class="chr-margin-left-10">Export CSV</span></a>
```

Comment:
The icon is still visible to the user but is not read to inaccessible users (with disability), more [here](http://stackoverflow.com/questions/31107040/whats-the-difference-between-html-hidden-and-aria-hidden-attributes)